packages/client/components/ActivityLibrary/ActivityLibrary.tsx

```diff
@@ -70,13 +94,18 @@ export const ActivityLibrary = (props: Props) => {
   }
 
   return (
-    <div className='flex h-full bg-white'>
-      <ActivityLibrarySideBar />
-      <div>
+    <div className='flex h-full w-full flex-col'>
```

Comment:
Was removing `bg-white` intentional here? White background is what's in the designs.

Reply:
not intentional, I was playing with some stuff and forgot to add it, thanks for catching it!